pyproject.toml

```
@@ -10,9 +10,12 @@ include = []
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-olas-operate-middleware = { git = "https://github.com/valory-xyz/olas-operate-app.git", rev = "caed60d6757f9fa477d9723fd2ec017413ce5d6f"}
+olas-operate-middleware = { git = "https://github.com/valory-xyz/olas-operate-app.git", rev = "caed60d6757f9fa477d9723fd2ec017413ce5d6f", python = ">=3.9,<3.12" }
```

this can be removed I guess
```suggestion
olas-operate-middleware = { git = "https://github.com/valory-xyz/olas-operate-app.git", rev = "caed60d6757f9fa477d9723fd2ec017413ce5d6f"}
```

---

removed